cogs/Logs.py

```diff
@@ -155,14 +172,19 @@ async def on_message_delete(self, message):
                 timestamp=datetime.now(),
             )
 
-            fields = [("Content", message.content, False)]
-
-            for name, value, inline in fields:
-                embed.add_field(name=name, value=value, inline=inline)
-
             embed.set_footer(
                 text=f"{message.author.name}#{message.author.discriminator}"
             )
+
+            n = 0
+            while len(message.content) > n:
+                embed.add_field(
+                    name="content",
+                    value=message.content[n : n + 1024],
```

Comment:
try to do a smart cuttoff and cutoff words and not chars

Reply:
its fine, they are just some logs